### PR TITLE
zh: update info on EKS and dashboard access

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

- Add more details about accessing TiDB with an nlb with internal scheme
- Link to info about using TiDB Dashboard
- Update info about TiDB Dashboard with Kubernetes

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1212
- Other reference link(s): 
pingcap/tidb-operator#3974
pingcap/tidb-operator#3941

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/deploy-on-aws-eks.md [hunk 1]
```diff
@@ -180,6 +180,10 @@ curl -O https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/
 
 如需了解更详细的配置信息或者进行自定义配置，请参考[配置 TiDB 集群](configure-a-tidb-cluster.md)
 
+> **注意：**
+>
+> 默认情况下，`tidb-cluster.yaml` 文件中的配置为 TiDB 的 LoadBalancer 设置了“内部”方案。这意味着 LoadBalancer 只能在 VPC 内部访问，而不能在外部访问。要通过 MySQL 协议访问 TiDB，你需要使用一个堡垒主机或使用 `kubectl port-forward`。如果你想在互联网上公开 TiDB，并且意识到这样做的风险，你可以在 `tidb-cluster.yaml` 文件中将 LoadBalancer 的方案从“内部”改为“面向互联网”。
```
Comment (DanielZhangQD) [inline zh/deploy-on-aws-eks.md:185]:
```suggestion
> 默认情况下，`tidb-cluster.yaml` 文件中 TiDB 服务的 LoadBalancer 配置为 "internal"。这意味着 LoadBalancer 只能在 VPC 内部访问，而不能在外部访问。要通过 MySQL 协议访问 TiDB，你需要使用一个堡垒机或使用 `kubectl port-forward`。如果你想在互联网上公开访问 TiDB，并且知晓这样做的风险，你可以在 `tidb-cluster.yaml` 文件中将 LoadBalancer 从 "internal" 改为 "internet-facing"。
```

File: zh/access-dashboard.md [hunk 2]
```diff
@@ -6,15 +6,19 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/access-dashboard/']
 
 # TiDB Dashboard 指南
 
-TiDB Dashboard 是 TiDB 4.0 专门用来帮助观察与诊断整个 TiDB 集群的可视化面板，你可以在 [TiDB Dashboard](https://docs.pingcap.com/zh/tidb/stable/dashboard-intro) 了解详情。本篇文章将介绍如何在 Kubernetes 环境下访问 TiDB Dashboard。
+> **警告：**
+>
+> PD 的 `/dashboard` 路径中提供了 TiDB Dashboard。除此以外的其他路径可能没有访问控制。
```
Comment (qiancai) [inline zh/access-dashboard.md:11]:
```suggestion
> TiDB Dashboard 位于 PD 的 `/dashboard` 目录。其他路径可能无法访问控制。
```

**Comment by Joyinqin** [comment]
@DanielZhangQD @qiancai comments are addressed. PTAL~

**Comment by qiancai** [comment]
@DanielZhangQD   PTAL.

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**